### PR TITLE
Add option to create different types of QR Codes

### PR DESCRIPTION
**Pull request by ursforrer**
#3 

The following functions were added
- Support for Vcard, Phonenumber, Link and Wifi Credentials
- Add cevi design to the whole application

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 # Stage 1: Compile and Build angular codebase
 # Use official node image as the base image
-FROM node:16 as build
+FROM node:20 as build
```
Comment (wp99cp) [inline frontend/Dockerfile]: I've some problems building the container with node:20.
The container gets stuck at the "npm ci" step. I had to switch the image to node:20-alpine, and then the installation worked without any issues. Is there anything against switching to alpine for that particular image? I don't see any problem with that.

File: frontend/src/app/design-switcher/components/design-switcher/design-switcher.component.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    cursor: pointer;
+  }
+  
+  .checkmark {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 25px;
+    width: 25px;
+    background-color: white;
```
Comment (wp99cp) [inline frontend/src/app/design-switcher/components/design-switcher/design-switcher.component.scss:28]: Can we change the default color to something with a little more contrast to the background or add a small border to the radio button? Depending on the display's contrast, the clickable area isn't visible very well.

**Review by wp99cp — CHANGES_REQUESTED** [review]
@ursforrer thank you very much for your work, the new layout/design looks amazing!

I've tested all QR-Code options, and they seem to work as expected. I've two small changes (see comments), afterwards, feel free to merge the PR. 

For now, deployment must be done manually, but I will take care of that as soon as the PR is merged.

**Comment by ursforrer** [comment]
@wp99cp Thanks for the code review. Changes were applied and the branch merged.

**Pull request merged**